src/lib/crypto/krb/etypes.c

```diff
@@ -167,6 +167,27 @@ const struct krb5_keytypes krb5int_enctypes_list[] = {
       krb5int_dk_cmac_prf,
       CKSUMTYPE_CMAC_CAMELLIA256,
       0 /*flags */ },
+
+    { ENCTYPE_AES128_CTS_HMAC_SHA256_128,
+      "aes128-cts-hmac-sha256-128", { "aes128-cts-sha256" },
```

Comment:
I'm still looking over the rest of the changes, but I would like us to use shorter aliases than "aes128-cts-sha256"; maybe "aes128-sha2" and "aes256-sha2"?